backend/routes/router.ts

```diff
@@ -96,6 +100,22 @@ router.put('/tag', tagValidator.updateTag, (req, res) => {
   tagController.updateTag(req, res)
 })
 
+// Advertisementのルーティング
+router.get('/advertisement',(req, res) => advertisementController.getAdvertisements(req, res))
+router.get('advertisemnet/:id', (req, res) => advertisementController.getAdvertisement(req, res))
```

Comment:
@preticure 
```suggestion
router.get('/advertisemnet/:id', (req, res) => advertisementController.getAdvertisement(req, res))
```

これでも行かないですか？